src/charm.py

```diff
@@ -75,6 +75,7 @@ def __init__(self, *args):
             db_user=None,
```

Comment:
Do you think an integration test that demonstrates this issue is worth it? Otherwise someone might just remove the code in the future again and there wouldn't be any test failure?

Reply:
Discussed with Weii, should be ok without an integration test since the user can't actually tell this problem is happening